demo/admin/src/products/future/generated/ManufacturersGrid.tsx

```diff
@@ -137,7 +138,24 @@ export function ManufacturersGrid(): React.ReactElement {
         },
         {
             field: "address_alternativeAddress_street",
-            headerName: intl.formatMessage({ id: "manufacturer.address.alternativeAddress.street", defaultMessage: "Alt-Street" }),
+            renderHeader: () => (
+                <Box style={{ display: "flex", alignItems: "center" }}>
+                    <GridColumnHeaderTitle label="Alt-Street" columnWidth={150}>
```

Comment:
I don't think you can hardcode the column width here since it's configurable in the generator

Reply:
The `columnWidth` is required here. I check for the given value or set the defaultWidth. See [here](https://github.com/vivid-planet/comet/pull/2362/files#diff-0bbff08da129614fa6814cb0a61ff3fd55a96b5dd51125081f025fa11f8845bcR487). I can move up the definition of `defaultMinWidth` to clarify it.

Reply:
Sorry, my bad.